Dockerfile-build

```diff
@@ -41,14 +41,18 @@ RUN yum install -y gcc gcc-c++ libgcc libstdc++ libgomp glibc
 # Git & gcc requirements
 RUN yum install -y libcurl-devel zlib-devel asciidoc xmlto wget make autoconf gettext gmp-devel mpfr-devel libmpc-devel
 
-#Install gcc from sources
-RUN wget https://0xdata-public.s3.amazonaws.com/gcc/gcc-5.3.0.tar.gz && \
-    tar zxf gcc-5.3.0.tar.gz && \
-    mkdir gcc-5.3.0-build && \
-    cd gcc-5.3.0-build && \
-    ../gcc-5.3.0/configure --program-suffix=-5.3 --enable-languages=c,c++ --disable-multilib && \
+RUN bash -c 'if [ `arch` = "ppc64le" ]; then \
```

Comment:
only if ppc64le are we going to use new gcc?

Reply:
In case of x86 `devtoolset-6` get installed and used to compile xgboost. It compiles with `libstdc++_static.a` so no extra libstdc++.so is required. Unfortunately there's no `devtools` pkg on ppc64le. There're more details in #dev-h2o-gpu channel.

Reply:
tx